Dockerfile-boot

```diff
@@ -1,9 +1,9 @@
-FROM golang:1.10-alpine as builder
```

Comment:
this was causing an error when building docker images
```
INFO[0078] Tag 1.10-alpine as builder not found in repository golang
```

Reply:
it works fine on my env and also in status-go repo. it must be something with your env

Reply:
indeed, i was running an old version of docker